src/model/decay.rs

```diff
@@ -86,7 +86,7 @@ impl DecayTracker {
         let mut decay_ratings = Vec::new();
         for i in 0..decay_weeks {
             // Increment time by 7 days for each decay application (this is for accurate timestamps)
-            let simulated_time = last_adjustment.timestamp + chrono::Duration::days(i * 7);
+            let simulated_time = last_adjustment.timestamp + chrono::Duration::days((i + 1) * 7);
```

Comment:
Will this actually fix the issue? seems like this will just start the timestamps a week later when the decay period is 4 months

Reply:
I guess this should probably incorporate DECAY_DAYS somehow